src/Benchmark/UdpStatSendingBenchmark.cs

```diff
@@ -21,7 +21,7 @@ public void Setup()
         {
             var config = new StatsDConfiguration
             {
-                // if you want verify that stats are recevied,
+                // if you want verify that stats are received,
```

Comment:
Maybe do a find-and-replace and see if there's any more of these as it's the second one today? 😃

Reply:
I've checked, there are no more.  But I changed the wording anyway; "you want verify" was not quite right.